auditbeat/docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '2'
+services:
+  beat:
+    build: ${PWD}/.
+    depends_on:
+      - proxy_dep
+    env_file:
+      - ${PWD}/build/test.env
+    working_dir: /go/src/github.com/elastic/beats/auditbeat
+    volumes:
+      - ${PWD}/..:/go/src/github.com/elastic/beats/
+    command: make
+
+  # This is a proxy used to block beats until all services are healthy.
+  # See: https://github.com/docker/compose/issues/4369
+  proxy_dep:
+    image: busybox
+    depends_on:
+      elasticsearch: { condition: service_healthy }
```

Comment:
@exekias In compose version 2 `depends_on` has to be an array and does not support the health check: https://docs.docker.com/compose/compose-file/compose-file-v2/#depends_on 2.1 is required for the above notation. But 2.1 is not supported by the module script. As there is no compose usage yet in auditbeat I will check if I can get away with using 2.1 here.

Reply:
If you want to use the uncoupled testing code you need to switch to something like this: https://github.com/elastic/beats/blob/master/metricbeat/docker-compose.yml

Where there is no `proxy_dep` and health checks are managed from testing framework

Reply:
It seems to work with 2.1 for now as there are no usage of the testing framework.